modules/apps/object/object-test/src/testFunctional/macros/ObjectField.macro

```diff
@@ -22,6 +22,31 @@ definition {
 			locator1 = "ObjectField#DOCUMENT_NAME",
 			value1 = ${folderName});
 	}
+	
+	@summary = "Default summary"
+	macro addTranslatableTextEntryViaUI {
+		LexiconEntry.gotoAdd();
+
+		Type(
+			key_fieldLabel = ${fieldLabel},
+			locator1 = "TextInput#GENERIC_TEXT_INPUT",
+			value1 = ${mainEntryValue});
+
+		Click(
+			key_buttonFieldEntryLanguage = ${buttonFieldEntryLanguage},
+			locator1 = "ObjectField#BUTTON_FIELD_ENTRY_LANGUAGE");
+
+		Click(
+			key_locale = ${locale},
+			locator1 = "Translation#DROPDOWN_MENU_ITEM");
+
+		Type(
+			key_fieldLabel = ${fieldLabel},
+			locator1 = "TextInput#GENERIC_TEXT_INPUT",
+			value1 = ${translatedEntryValue});
+
+		Button.clickSave();
```

Comment:
Generally, I prefer not to add the navigation macro and save function within the same macro to configure something. The reason why is that it doesn't allow use to re-use the configuration macro later on in another context. For now it's fine but we should consider the modularity of our macro design in the future.